python/tests/helpers/lsp/responses/semantic_tokens_response.py

```diff
@@ -0,0 +1,10 @@
+from tests.helpers.lsp.responses.lsp_response import LspResponse
+
+
+class SemanticTokensResponse(LspResponse):
+    def get_default_data(self) -> list[int]:
+        return self.response.get("data", [])
+
+    def get_semantic_tokens(self) -> list[list[int]]:
+        data = self.get_default_data()
+        return [data[i:i + 5] for i in range(0, len(data), 5)]
```

Comment:
Out of curiosity, what's the '5' for? Should this be dynamic or configurable?

Reply:
Each semantic token is represented by 5 integers - the line number relative to the last token, the start character relative to the last token, the length of the token, the index corresponding to the token type in the `SemanticTokensLegend`, and a bitmask of all corresponding token modifiers in the `SemanticTokensLegend`.

For reference see the section on [Integer Encoding for Tokens](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_semanticTokens:~:text=relative%27%3B-,Integer%20Encoding%20for%20Tokens,-On%20the%20capability)